transformato/annihilation.py

```diff
@@ -325,6 +333,11 @@ def _mutate_to_common_core(
                 if atom_idx in hydrogens:
                     # is already mutated in the previouse step
                     continue
+                elif self.psf1["waterbox"].atoms[atom_idx].type == "LPH":
```

Comment:
LP should not have  lj interaction, so they do not need to be turned off in one seperate step. **ATTENTION: `self.psf1` needs to be changed to be flexible for RBFE**